CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ title: Changelog
 * `<cc-toggle>`: 
   * rename `--cc-text-transform` to `--cc-toggle-text-transform` (BREAKING CHANGE),
   * add CSS Custom Prop to customize `border-radius`, `font-weight`.
+* `default-theme` and most of the components:
```

Comment:
I would rename this to something like `Introduce a public theme based on a CSS files`, WDYT?